app/controllers/application_controller.rb

```diff
@@ -414,6 +414,11 @@ def webpack_server
     @dev_server ||= "#{request.protocol}#{request.host}:#{port}"
   end
 
+  def compute_resource_error(listing, error)
```

Comment:
i dont think it make sense to create a compute resource specific error handling in the application controller....